test/library/options-test.ts

```diff
@@ -37,7 +37,6 @@ describe('options test', () => {
 		expect(store.options.draft_polli_ratelimit_headers).toEqual(true)
 	})
 
-	// TODO: Update in v8.
```

Comment:
![never gonna give you up](https://i258.photobucket.com/albums/hh253/jimifunguzz/rick-astley-dancing.gif)